detectors/aws/eks/detector.go

```diff
@@ -55,8 +55,8 @@ var _ resource.Detector = (*resourceDetector)(nil)
 // Compile time assertion that eksDetectorUtils implements the detectorUtils interface.
 var _ detectorUtils = (*eksDetectorUtils)(nil)
 
-// is this going to stop working with 1.20 when Docker is deprecated?
-var containerIDRegex = regexp.MustCompile(`^.*/docker/(.+)$`)
+// Updated regex to support both Docker and Kubernetes/containerd cgroup formats
```

Comment:
Comments shouldn't be about things that have been updated (GIT has history for that), but actual issues.